.phraseapp.yml

```diff
@@ -0,0 +1,17 @@
+phraseapp:
+  project_id: d2a63bc2ab1c63e5732b6e77fff1fbd1
+  file_format: strings
+  push:
+    sources:
+    - file: Stripe/Resources/Localizations/en.lproj/Localizable.strings
```

Comment:
This is likely not the correct file for Android, right? A bad copy and paste from iOS?

Reply:
yeah sorry, this PR was only for pull!